common/lib/capa/capa/templates/status_span.html

```diff
@@ -0,0 +1,16 @@
+<%page expression_filter="h" args="status, status_id='', hide_correctness=False"/>
+
+% if status_id == '':
+<span class="status ${'' if hide_correctness == True else status.classname}"
+      data-tooltip="${'' if hide_correctness == True else status.display_tooltip}">
+% else:
+<span class="status ${'' if hide_correctness == True else status.classname}" id="status_${status_id}"
+      data-tooltip="${'' if hide_correctness == True else status.display_tooltip}">
+% endif
+  % if hide_correctness == False:
+    <span class="sr">
```

Comment:
Can you put this on a single line (lines 11-13) to remove extra spaces that end up in the page-level sr region?